techniques/systemSettings/userManagement/sudoParameters/3.2/sudoParameters.st

```diff
@@ -17,12 +17,14 @@
 #####################################################################################
 
 ##########################################################################
-# Sudo configuration Technique                                                  #
+# Sudo configuration Technique                                           #
 # ---------------------------------------------------------------------- #
 # Objective : Configure /etc/sudoers according to the given parameters   #
 ##########################################################################
 
```

Comment:
Could you add a comment to explain the general behavior (tmp file vs? audit mode)?